n_threads: 3                     # number of threads to use.
r_min_square: 4                   # minimum point distance square.
r_max_square: 6400                # maximum point distance square.
n_bins: 100                      # number of radial bins.
n_segments: 360                  # number of radial segments.
max_dist_to_line: 0.25           # maximum vertical distance of point to line to be considered ground.
sensor_height: 1.79              # sensor height above ground.
max_slope: 0.2                   # maximum slope of a ground line.
max_fit_error_square: 0.05       # maximum error square of a point during line fit.
long_threshold: 3.0              # distance between nearest points on a line after which they are considered far from each other.
max_long_height: 0.2             # maximum height change to previous point in long line.
max_start_height: 0.2            # maximum difference to estimated ground height to start a new line.
line_search_angle: 0.1           # how far to search in angular direction to find a line [rad].
visualize: false                 # visualize segmentation result - USE ONLY FOR DEBUGGING
ground_max_height: 0.5           # ground max height
ground_value: 0.05               # ground mean height